md/release-notes.md

```diff
@@ -9,7 +9,16 @@
 
 ## Technology updates
 
+<a id="feature-removals"/>
 
+## Feature removals
+
+<a id="6.x-form"/>
+
+### 6.x forms based on GWT technology removed.
+Studio forms based on Google Web Toolkit (GWT) technology are not supported anymore, starting with Bonita 7.8. They have been removed from Bonita Studio. Importing a .bos will not import such forms.
+Cloning a Git repository or migrating a SVN repository will remove such forms and their related content (validators, look'n'feels, forms and widgets templated).  
+Make sure you have used the newer mechanism in Bonita Studio older than 7.8 to replace your forms, using pool and task contracts as well as the UI Designer, as [documented here](contracts-and-contexts.md). The case overview page should also be replaced if you have customized it.
```

Comment:
What about that:
"Make sure you have replaced such forms by forms created with more recent technologies and newer concepts, offered since Bonita 7.0: [UI Designer](ui-designer-overview.md) and [contract and context](contracts-and-contexts.md), before you start Bonita Studio 7.8 or migrate your production to Bonita 7.8."

Reply:
Ok, and maybe explicit a little "more recent technologies"...